hog-contest/logger.py

```diff
@@ -0,0 +1,15 @@
+from collections import deque
+from datetime import datetime
+
+MAX_LOG_LEN = 100
+
+logs = deque(maxlen=100)
```

Comment:
any reason for not using the variable defined above? i can't tell if it's supposed to be the same but it seems like it is